javascript/javascript1/week2/homework5.js

```diff
@@ -0,0 +1,45 @@
+// Candy helper optional
+
+const boughtCandyPrices = [];
+
+const addCandy = (candyType, weight) => {
+  candyType.toLowerCase();
+
+  switch (candyType) {
+    case (candyType = "sweet"):
+      boughtCandyPrices.push(weight * 0.5);
+      break;
+    case (candyType = "chocolate"):
+      boughtCandyPrices.push(weight * 0.7);
+      break;
+    case (candyType = "toffee"):
+      boughtCandyPrices.push(weight * 1.1);
+      break;
+    case (candyType = "chewing-gum"):
+      boughtCandyPrices.push(weight * 0.03);
+      break;
+  }
```

Comment:
Great, it works! The assignment asks to use a for loop but it seems you have grasped the logic.

How could you re-write a loop so you don't have to repeat yourself `boughtCandyPrices.push(weight *.....`?